library_generation/test/model/config_change_unit_tests.py

```diff
@@ -0,0 +1,224 @@
+                ],
+                ChangeType.LIBRARIES_ADDITION: [
+                    LibraryChange(
+                        changed_param="test",
+                        latest_value="test",
+                        library_name="new-library",
+                    ),
+                ],
+            },
+            baseline_config=ConfigChangeTest.__get_a_gen_config(
+                googleapis_commitish="277145d108819fa30fbed3a7cbbb50f91eb6155e"
+            ),
+            latest_config=ConfigChangeTest.__get_a_gen_config(
+                googleapis_commitish="8984ddb508dea0e673b724c58338e810b1d8aee3",
+                libraries=[
+                    ConfigChangeTest.__get_a_library_config(
+                        library_name="gke-backup",
+                        gapic_configs=[
+                            GapicConfig(proto_path="google/cloud/gkebackup/v1")
+                        ],
+                    ),
+                ],
+            ),
+        )
+
+        self.assertEqual(
+            ["a-library", "gke-backup", "new-library"],
+            sorted(config_change.get_changed_libraries()),
+        )
+
+    def test_get_qualified_commits_success(self):
+        config_change = ConfigChange(
+            change_to_libraries={},
+            baseline_config=ConfigChangeTest.__get_a_gen_config(
+                googleapis_commitish="277145d108819fa30fbed3a7cbbb50f91eb6155e"
+            ),
+            latest_config=ConfigChangeTest.__get_a_gen_config(
+                googleapis_commitish="8984ddb508dea0e673b724c58338e810b1d8aee3",
+                libraries=[
+                    ConfigChangeTest.__get_a_library_config(
+                        library_name="gke-backup",
+                        gapic_configs=[
+                            GapicConfig(proto_path="google/cloud/gkebackup/v1")
+                        ],
+                    ),
+                    ConfigChangeTest.__get_a_library_config(
+                        library_name="aiplatform",
+                        gapic_configs=[
+                            GapicConfig(proto_path="google/cloud/aiplatform/v1beta1")
+                        ],
+                    ),
+                    ConfigChangeTest.__get_a_library_config(
+                        library_name="network-management",
+                        gapic_configs=[
+                            GapicConfig(proto_path="google/cloud/networkmanagement/v1"),
+                            GapicConfig(
+                                proto_path="google/cloud/networkmanagement/v1beta1"
+                            ),
+                        ],
+                    ),
+                ],
+            ),
+        )
+        qualified_commits = config_change.get_qualified_commits()
+        self.assertEqual(3, len(qualified_commits))
+        self.assertEqual({"gke-backup"}, qualified_commits[0].libraries)
+        self.assertEqual(
+            "b8691edb3f1d3c1583aa9cd89240eb359eebe9c7",
```

Comment:
nit: can we use variable names for these commit hashes? This is just in order to make the test more self-explanatory.